SYCL.make

```diff
@@ -0,0 +1,11 @@
+
+sycl-stream: main.cpp SYCLStream.cpp SYCLStream.sycl
+	$(CXX) -O3 -std=c++11 -DSYCL main.cpp SYCLStream.cpp -include SYCLStream.sycl -lComputeCpp -lOpenCL -o $@
+
+
+SYCLStream.sycl: SYCLStream.cpp
+	compute++ SYCLStream.cpp -sycl -no-serial-memop -O2 -emit-llvm -c
```

Comment:
If computecpp_info is on the path (as compute++ seems to be) you can just query it for the device compiler flags:
```
$ computecpp_info  --dump-device-compiler-flags
-O2 -mllvm -inline-threshold=1000 -sycl -intelspirmetadata -emit-llvm
```
which you can use then for the makefile